aries_cloudagent/protocols/issue_credential/v2_0/routes.py

```diff
@@ -1212,12 +1219,13 @@ async def credential_exchange_send_bound_request(request: web.BaseRequest):
             except StorageNotFoundError as err:
                 raise web.HTTPNotFound(reason=err.roll_up) from err
 
-            connection_id = cred_ex_record.connection_id
+        connection_id = cred_ex_record.connection_id
+        async with profile.session() as session:
```

Comment:
I don't think it makes sense to open a new session in this case. It's just assigning a value before going to DB again.

Reply:
Fixed